react-challenges/github-finder/devashmit/README.md

```
@@ -0,0 +1,49 @@
+## 🚀 How to Run
+
+```bash
+npm install
+npm run dev
+```
+
+Open [http://localhost:5173](http://localhost:5173)
+
+## 🗂 Project Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)